wavenet.py

```diff
@@ -171,7 +188,7 @@ def _create_network(self, input_batch):
 
             # We skip connections from the outputs of each layer, adding them
             # all up here.
-            total = sum(outputs)
+            total = tf.add_n(outputs)
```

Comment:
Does this bring performance improvements?
The rationale behind using `sum` was that TensorFlow could perform the sum in parallel while calculating the layers, and didn't have to keep all skipped outputs around until the end.
Then again, the various sum operations might allocate a lot of memory.